.gitignore

```diff
@@ -8,4 +8,3 @@ _site
 .tweet-cache
 Gemfile.lock
 vendor
-
```

Comment:
Can you revert these changes?

Reply:
Reverted! Not sure how that happened there. Forgot I could just git rebase those commits away